workspaces/sonarqube/.changeset/version-bump-1-42-3.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage-community/plugin-sonarqube': minor
+'@backstage-community/plugin-sonarqube-backend': minor
+'@backstage-community/plugin-sonarqube-react': minor
```

Comment:
arguable whether this is breaking or not, but happy to leave as minor if that's the most sensible

Reply:
for community-plugins every Backstage version bump is treated as minor